getting_started/first_3d_game/09.adding_animations.rst

```diff
@@ -90,27 +95,27 @@ Select the ``Character`` and in the *Inspector* expand the *Transform* section.
 
 For this tutorial, just create RESET Track(s) which is the default choice
 
-Two tracks appear in the editor with a diamond icon representing each keyframe.
+Two tracks appear in the editor with a small inline icon representing each keyframe.
```

Comment:
This comment is more about the pictures. Are you doing something different with how you create your keyframes? I just tested this in the finished demo project by deleting the animation and re-creating it. The initial icon is a diamond for the keyframes. It doesn't change into a square until you adjust easing later in the tutorial.

Reply:
Oof. This was a bad assumption on my part. After finally getting the animation to work right, I just deleted keyframes and took the screenshots showing the correct positions. I had assumed that the icon changed aesthetically in Godot 4, but re-creating new keyframes I correctly see the diamonds that change to a square when applying easing.

This is my mistake for being new-ish to the domain (I'm usually working enterprise software). I'll go back through the steps and update the screenshots with corrected markers and drop the text changes.

- [ ] Update screenshots, taking them before easing is applied where appropriate
- [ ] Remove language changes about keyframe markers